src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -259,14 +262,19 @@ private void updatePR(GHPullRequest ghpr, GHIssueComment comment, boolean isWebh
 
             synchronized (this) {
                 boolean wasUpdated = setUpdated(updatedDate);
-
+                
```

Comment:
Trailing spaces.  I'm only going to comment this once.  But anywhere there's trailing spaces in this PR should be removed.